website/docs/docusaurus-core.md

```diff
@@ -298,6 +298,25 @@ const MyComponent = () => {
 };
 ```
 
+### `onRouteUpdate({location})`
+
+Set the page field to current location on the default tracker, such that subsequent hits on this page are attributed to this page.
```

Comment:
This is copy-paste from [here](https://github.com/facebook/docusaurus/blob/1f00d15c7439fdbfb37d57f0cafdaedb2b34ef41/packages/docusaurus-plugin-google-analytics/src/analytics.js). This is definitely not suitable for docs, because it is completely incomprehensible for end-users.

Reply:
@lex111 Let me do a bit more homework on this. 👍

Reply:
@lex111 I have done some changes, based on end-user perspective. Please Review